source/pattern-lab.js

```diff
@@ -25,7 +25,7 @@ const settings = {
 };
 
 // Let's just execute everything and pass in $(document), settings
-_.forEach(designSystem, (component, name) => {
+_.forEach(ds, (component, name) => {
```

Comment:
Agree shorthand might be better here, but are we sure `ds` is descriptive enough to understand for new people looking at this file?

Reply:
Potentially painful for blind refactoring. thunderSnow?

Reply:
`ds` is explicitly aliased from the `design-system.js` component. However, I'll be looking into a better way of exporting all patterns fro design-system, since it currently exports a `designSystem` object causing linter warnings if we attempt to alias locally as the same name.

Reply:
Captured here: I've got an idea for a cleaner, less object-y approach to components: 

https://github.com/phase2/pattern-lab-starter/issues/160